collect_app/src/main/java/org/odk/collect/android/activities/BaseActivity.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2018 Shobhit Agarwal
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.odk.collect.android.activities;
+
+import android.os.Bundle;
+import android.support.annotation.CallSuper;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.view.Menu;
+
+import org.odk.collect.android.utilities.ThemeUtils;
+
+public abstract class BaseActivity extends AppCompatActivity {
```

Comment:
We already have a base activity it's called CollectAbstractActivity so you can use it.

Reply:
Thanks for the suggestion. Things look much better now :smiley: